GooglePlayInstant/Editor/CommandLineBuilder.cs

```diff
@@ -74,7 +74,8 @@ public static string GetApkPath()
 
         private static void SetTargetArchitectures()
         {
-#if UNITY_2018_1_OR_NEWER
+            // Unity 2017.4.16 added support for ARM64. There is no UNITY_2017_4_16_OR_NEWER, so we make a best effort.
+#if UNITY_2018_1_OR_NEWER || UNITY_2017_4_16 || UNITY_2017_4_17 || UNITY_2017_4_18 || UNITY_2017_4_19 || UNITY_2017_4_20 || UNITY_2017_4_21 || UNITY_2017_4_22 || UNITY_2017_4_23 || UNITY_2017_4_24 || UNITY_2017_4_25 || UNITY_2017_4_26 || UNITY_2017_4_27 || UNITY_2017_4_28 || UNITY_2017_4_29
```

Comment:
Should we define a UNITY_2017_4_17_OR_NEWER that could be reused between this line and the one in AppBundlePublisher?

Reply:
- As far as I can tell one can't `#define ` across files. The way to have preprocessor directives cross files is through .csproj files or Unity's "Scripting Define Symbols".
- In the App Bundle case we need 2017.4.17+ but not any 2018.1 or 2018.2, so "OR_NEWER" would be confusing. (Normally OR_NEWER should include 2018.1)
- P.S. I avoid defining preprocessor directives starting with "UNITY_" and instead use "PLAY_INSTANT_" to avoid surprises.

Reply:
I see, those are all good points. I'm fine leaving it how it is then.